R/prepare_ictr_product.R

```diff
@@ -54,7 +54,7 @@ rename_ictr_product <- function(data) {
       ICTR_risk_category = "risk_category",
       input_name = "exchange_name",
       input_unit = "exchange_unit_name",
-      input_isic_name = "isic_4digit_name_ecoinvent"
+      isic_4digit_name = "isic_4digit_name_ecoinvent"
```

Comment:
The title of the PR is misleading:

* This change affects `*ictr*()` (now `profile_emissions_upstream()`), so it is not limited to the `*sector*()` functions.
* This change renames a column, so it's not limited to adding a column.

Also, why do we remove the `input_` prefix on this particular column? Although I would love to remove that prefix I think we should either remove it from everywhere or  nowhere.